tests/numpy/test_arithmetic.py

```diff
@@ -78,6 +78,50 @@ def check_bop(_name, _np_a, _np_b):
                 check_bop(name, np_a, np_b)
 
 
+def test_bitwise_error(nps_app_inst):
+    assert nps_app_inst is not None
+
+    uops, bops = numpy_utils.ufunc_op_signatures()
+    uops_names, bops_names = [_op[0] for _op in uops], [_op[0] for _op in bops]
+
+    def check_bitwise_error(_name, error):
+        np_ufunc = np.__getattribute__(_name)
+        ns_ufunc = nps.__getattribute__(_name)
+
+        _np_a = np.random.randn(2, 2)
+        _np_b = np.random.randn(2, 2)
+
+        _ns_a = nps.array(_np_a)
+        _ns_b = nps.array(_np_b)
+        message = ""
+
+        try:
+            if _name in uops_names:
```

Comment:
Why do we need to perform this test? If the names of the operations are different, can we pass the numpy names as well as the nums names?